api.planx.uk/webhooks/paymentRequestEvents.ts

```diff
@@ -0,0 +1,114 @@
+import { addDays } from 'date-fns';
+import { Request, Response, NextFunction } from 'express';
+
+import { createScheduledEvent } from '../hasura/metadata';
+import { DAYS_UNTIL_EXPIRY } from '../saveAndReturn/utils';
```

Comment:
worth pointing out that this aligns the payment request "validity period" to `lowcal_sessions` - both at 28 days. may make sense to treat uniquely in the future, but this felt straightforward for now until we have guidance that suggests another time period